test/MusicStore.Test/ShoppingCartControllerTest.cs

```diff
@@ -27,6 +30,8 @@ public ShoppingCartControllerTest()
                       .AddInMemoryStore()
                       .AddDbContext<MusicStoreContext>();
 
+            services.AddMvc();
```

Comment:
@Eilon this view does not have a form. The click even of the remove link is hijacked to perform a POST. So the ajax code sends the xsrf token along with the request. I remember annotating the action with [ValidateAntiForgery..] did not work for some reason and I saw this way of validation in one of the web api samples in asp.net site. But I think I can replace it with the attribute as long as I POST the token in the expected format. I'll give it a try.